### ISSUE TITLE
Problems with accessing google authentication

### ISSUE THREAD

**Issue by HMCNJ** [MRCIEU/TwoSampleMR#113]
We try to extract_instrument from MR Base database, we failed to get a link to my Google account and the errors as follows:
`a<-extract_instruments(outcomes=324)
Requesting default values. Extracting from pre-clumped data
0 out of 1 requested outcomes have pre-calculated instruments. 
Extracting data from 1 GWAS(s)
and performing LD clumping
 [>] 1 of 1
Error: Cannot read token from alleged .rds file:
mrbase.oauthError in readRDS(token_path): error reading from connection'

**Comment by luochihiro** [comment]
I have the same question

**Comment by SunnyWit** [comment]
did you resolve this problem? i also meet this question

**Comment by explodecomputer** [comment]
hey it might be that you have an empty file, try removing the mrbase.oauth file and re-authenticating

**Comment by SunnyWit** [comment]
yes,I&nbsp;reauthenticate several times,but&nbsp;&nbsp; mrbase.oauth file is always empty



---Original---

**Comment by explodecomputer** [comment]
Sorry I'm not sure why this is happening. I would just check that the mrbase.oauth file in the working directory is definitely being removed (e.g. check the time stamp of the newly created one), and if problems still persist then avoid using authentication by:

```
a<-extract_instruments(outcomes=324, access_token=NULL)
```

**Comment by HMCNJ** [comment]
you can only extract the avaliable trait id.
ao<-available_outcomes(access_token = NULL)

At 2019-11-15 18:51:55, "gibran hemani" <notifications@github.com> wrote:


Sorry I'm not sure why this is happening. I would just check that the mrbase.oauth file in the working directory is definitely being removed (e.g. check the time stamp of the newly created one), and if problems still persist then avoid using authentication by:

a<-extract_instruments(outcomes=324, access_token=NULL)

**Comment by HMCNJ** [comment]
you can only extract the avaliable trait id. 
ao<-available_outcomes(access_token = NULL)

**Comment by HMCNJ** [comment]
> did you resolve this problem? i also meet this question

you can only extract the avaliable trait id. 
ao<-available_outcomes(access_token = NULL)
